content/docs/command-reference/check-ignore.md

```diff
@@ -0,0 +1,85 @@
+# check-ignore
+
+Check whether any given files or directories are excluded from DVC
+due to the patterns found in [`.dvcignore`](/doc/user-guide/dvcignore).
+
+## Synopsis
+
+```usage
+usage: usage: dvc check-ignore [-h] [-q | -v] [-d] [-n]
+                               targets [targets ...]
+
+positional arguments:
+  targets        File or directory paths, or ignore patterns to check
+```
+
+## Description
+
+This helper command checks whether the given `targets` are ignored by DVC
+according to the [`.dvcignore` file](/doc/user-guide/dvcignore) (if any). The
+ones that are indeed ignores are printed back.
+
+## Options
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+- `-d`, `--details` - show the exclude pattern together with each target path.
+
+- `-n`, `--non-matching` - show the target paths which don’t match any pattern.
+  Only usable when `--details` is also employed
```

Comment:
Per https://github.com/iterative/dvc.org/pull/1629#pullrequestreview-457172831:

```suggestion
- `-d`, `--details` - show the exclude pattern together with each target path.

- `-n`, `--non-matching` - show the target paths which don’t match any pattern.
  Only usable when `--details` is also employed

- `-h`, `--help` - prints the usage/help message, and exit.

- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
  problems arise, otherwise 1.

- `-v`, `--verbose` - displays detailed tracing information.
```

Committing.